crow/setup3.py

```diff
@@ -18,15 +18,15 @@
 except:
   eigen_flags = ""
 include_dirs=['include/distributions','include/utilities','contrib/include']
-if eigen_flags.startswith("-I"):
+if eigen_flags.startswith(b"-I"):
```

Comment:
The line above should probably be changed to:
```
eigen_flags = subprocess.check_output(["./scripts/find_eigen.py"]).decode("ascii")
```
so that eigen_flags is ascii.